### ISSUE TITLE
Undefined property: stdClass::$error

### ISSUE THREAD

**Issue by pixeline** [cjrasmussen/BlueskyApi#8]
This error points to line: /vendor/cjrasmussen/bluesky-api/src/BlueskyApi.php:153

it should be

```php
		$data = $this->request('POST', 'com.atproto.server.createSession', $args);

		if (!empty($data->error)) {
			throw new RuntimeException($data->message);
		}
```
I'm not even sure $data->error will actually ever be created, as $data is the result of json_decode().

Thanks!

**Comment by cjrasmussen** [comment]
Good catch.  Fixed in v2.0.1.

As for `$data->error` ever being created...  If I recall correctly, this was specifically related to #6 and `com.atproto.server.createSession` returning an error value when hitting rate limits.

**Comment by jaybeaton** [comment]
@cjrasmussen

You'll need to make this same change in `BlueskyApi::refreshSession()`.

**Comment by cjrasmussen** [comment]
@jaybeaton 

Thanks for the heads-up.  Resolved in v2.0.2.
